CMakeLists.txt

```diff
@@ -231,15 +228,23 @@ if(UNIX)
         # linux
         IF(CPACK_GENERATOR STREQUAL "RPM")
             set(CPACK_RPM_PACKAGE_LICENSE "GPLv2")
-            set(CPACK_RPM_PACKAGE_REQUIRES "protobuf, qt5-qttools, qt5-qtsvg, qt5-qtmultimedia")
+            IF(QT6_FOUND)
+                SET(CPACK_RPM_PACKAGE_REQUIRES "protobuf, qt6-qttools, qt6-qtsvg, qt6-qtmultimedia")
+            ELSEIF(Qt5_FOUND)
```

Comment:
if neither of these is found an error happens, doing the double check here seems weird